osu.Game/Online/API/Requests/Responses/CommentBundle.cs

```diff
@@ -26,7 +26,7 @@ public List<Comment> Comments
                         {
```

Comment:
this is very weird logic. there's no need to `ForEach` since there should be either zero or one match. Try using `FirstOrDefault` instead?